your-rents-core/src/main/java/com/yourrents/core/YRNotFoundException.java

```diff
@@ -0,0 +1,42 @@
+package com.yourrents.core;
+
+/*-
+ * #%L
+ * Your Rents Core
+ * %%
+ * Copyright (C) 2019 - 2021 Your Rents Team
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+public class YRNotFoundException extends RuntimeException {
```

Comment:
Qual è il vantaggio di chiamarla YRNotFoundException? Non basta NotFoundException?

Reply:
Sure, you’re right, I’ll change it